platform_versioned_docs/version-24.1.0/launch/launchpad.mdx

```diff
@@ -5,95 +5,96 @@ date: "21 Apr 2023"
 
-The remaining fields of the pipeline parameters form will vary for each pipeline, dependent on the parameters specified in the pipeline schema. When you have filled the necessary launch form details, select **Launch**. The **Runs** tab shows your new run in a **submitted** status at the top of the list. Select the run name to navigate to the [**View Workflow Run**](../monitoring/run-details.mdx) page and view the configuration, parameters, status of individual tasks, and run report.
+- **Workflow run name**: A unique identifier for the run, pre-filled with a random name. This can be customized.
 
-:::note
-For more information on relaunch and resume, see [Cache and resume](./cache-resume.mdx).
-:::
+- **Labels**: Assign new or existing [labels](../labels/overview.mdx) to the run.
+
+- **Compute environment**: The [compute environment](../compute-envs/overview.mdx) where the run will be launched.
+
+- **Work directory**: The cloud storage or file system path where pipeline scratch data is stored. Seqera will create a scratch sub-folder if only a cloud bucket location is specified. Use file system paths for local or HPC compute environments.
 
-**Quick launch an unconfigured pipeline**
+  :::note
+  The credentials associated with the compute environment must have access to the work directory if it resides in cloud storage.
+  :::
 
-1. Select **launch a run without configuration** below the Launchpad heading.
-1. On the **Launch** form, enter an optional **Workflow run name** (or keep the randomly assigned default) and assign optional labels to the run.
-1. Select a **Compute environment** from the available options. See [Compute environments](../compute-envs/overview.mdx) to learn how to create an environment for your preferred execution platform.
-1. Enter a repository URL for the **Pipeline to launch** (e.g., `https://github.com/nf-core/rnaseq.git`).
+### Run parameters 
 
-    :::note
-    Nextflow pipelines are Git repositories that can reside on any public or private Git-hosting platform. See [Git integration](../git/overview.mdx) in the Seqera docs and [Pipeline sharing](https://www.nextflow.io/docs/latest/sharing.html) in the Nextflow docs for more details.
-    :::
+There are three ways to enter **Run parameters** prior to launch:
+
+- The **Input form view** displays form fields to enter text, select attributes from dropdowns, and browse input and output locations with [Data Explorer](../data/data-explorer.mdx).
+- The **Config view** displays a raw schema that you can edit directly. Select JSON or YAML format from the **View as** dropdown.
+- **Upload params file** allows you to upload a JSON or YAML file with run parameters.
+
+Seqera uses a `nextflow_schema.json` file in the root of the pipeline repository to dynamically create a form with the necessary pipeline parameters. All pipelines contain at least input and output parameters:
```

Comment:
is this true? do all pipeline with a schema have input and output parameters or is it an nf-core convention that can be skipped?